packages/docs/node.md

```diff
@@ -0,0 +1,65 @@
+---
+order: -50
+icon: package
+visibility: hidden
+---
+
+# Node
+
+The Bitcoin Computer Node is a vanilla Bitcoin (or Litecoin) node enhanced with the following capabilities:
+
+**Pre-computed Balances**. All balances are pre-computed during synchronization (on a vanilla node it takes ca 10 mins to initially compute the balance of a given address, which leads to a bad UX)
+
+**Recent Transactions**. It can return the recent sent a received transactions for a given address (this is not supported on vanilla nodes)
+
+**Optimized for Development**. On regtest, the halving period is increased to infinity. This has the advantage that a large number of tests can be run without having to restart the node.
+
+**Multi-Chain**. Supports BTC and LTE
+
+**Multi-Platform**. Multiplatform Docker images work on all major architectures (todo: list architectures) 
+
+**Easy to Use**. No compilation needed. You can start the node in a single line of code.
+
+**Easy to Configure**. Allow and deny access to RPC endpoints using a regular expression
+
+## Configuration
+
+```bash
```

Comment:
We can copy the latest version of the .env.example, or even add a link here to the github file for the .env.example